spec/features/search_page_spec.rb

```diff
@@ -32,13 +32,12 @@
     old_dataset = DatasetBuilder.new
                     .with_title('Old Interesting Dataset')
                     .with_name('old-dataset')
-                    .last_updated_at('2014-07-24T14:47:25.975Z')
+                    .public_updated_at(Time.now - 1)
```

Comment:
Could you explain what the `- 1` is for?

Reply:
To make it an older date than the new dataset's public_update_at.

Reply:
Oops, I see - thanks :-)